CromIAM/src/test/scala/cromiam/sam/SamClientSpec.scala

```diff
@@ -76,19 +88,14 @@ class SamClientSpec extends AsyncFlatSpec with Matchers with BeforeAndAfterAll
     )
   }
 
-  it should "return sam errors while checking collections" in {
```

Comment:
By deleting this test are you saying it should *never* return sam errors?

Reply:
Related to the not-really-a-Mock discussion, I believe this Spec was testing the "Mock" method, and not actually reaching any part of the main-code.